bundle/edu.gemini.sp.vcs/src/test/scala/edu/gemini/sp/vcs/diff/TestEnv.scala

```diff
@@ -23,18 +25,34 @@ case class TestPeer(odb: IDBDatabaseService, server: VcsServer, service: Princip
   def vcs(p: Principal): Vcs =
     new Vcs(VcsAction(Set(p)), server, _ => service(p))
 
-  val staffVcs: Vcs = vcs(StaffPrincipal.Gemini)
+  val superStaffVcs: Vcs = vcs(StaffPrincipal.Gemini)
 
   def shutdown(): Unit = odb.getDBAdmin.shutdown()
 
+  def spDataObject: SPProgram = prog.getDataObject.asInstanceOf[SPProgram]
+
   // Lookup the test program
   def prog: ISPProgram = odb.lookupProgram(TestEnv.Key)
 
+  def set(update: SPProgram => Unit): Unit =
+    spDataObject <| update |> prog.setDataObject
```

Comment:
We will soon get to the point where you can feed any ASCII art into the Scala compiler and out comes .. a program. ;)